packages/reports/addon/templates/components/navi-date-picker.hbs

```diff
@@ -1,22 +1,22 @@
-{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{! Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
```

Comment:
Do you have any hourly test for `navi-date-picker`?

Reply:
Added some now!